packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -7,7 +7,7 @@ import type { UrlWithParsedQuery } from 'url';
 import url from 'url';
 import querystring from 'querystring';
 import type { AxiosError, AxiosRequestConfig, AxiosResponse } from 'axios';
-import axios from 'axios';
+import { axiosInstance as axios } from '@nangohq/shared/lib/axiosInstance/index.js';
```

Comment:
import is wrong, it should be export from shared index.ts

Reply:
fixed